substrafl/remote/register/register.py

```diff
@@ -62,7 +62,7 @@
 ENV PATH="/home/user/venv/bin:$PATH" VIRTUAL_ENV="/home/user/venv"
 
 # install dependencies
-RUN python{python_version} -m pip install -U pip
+RUN python{python_version} -m pip install -U pip && pip install -U setuptools>=70.0.0
```

Comment:
Or we can just uninstall it...

Reply:
Not sure if it's not used when building wheels for some more exotic dependencies... :/